packages/@coorpacademy-app-review/src/index.tsx

```diff
@@ -81,26 +80,60 @@ const mapStateToSlidesProps = (state: StoreState): SlidesViewStaticProps | null
   }
 
   return {
-    headerProps: {
+    header: {
       mode: '__revision_mode',
       skillName: '__agility',
       onQuitClick: (): void => {
         // eslint-disable-next-line no-console
         console.log('onQuitClick');
       },
       'aria-label': 'aria-header-wrapper',
-      closeButtonAriaLabel: 'aria-close-button'
+      closeButtonAriaLabel: 'aria-close-button',
+      steps: [
+        {
+          icon: 'no-answer',
+          current: true,
+          value: '1'
+        },
+        {
+          icon: 'no-answer',
+          current: false,
+          value: '2'
+        },
+        {
+          icon: 'no-answer',
+          current: false,
+          value: '3'
+        },
+        {
+          icon: 'no-answer',
+          current: false,
+          value: '4'
+        },
+        {
+          icon: 'no-answer',
+          current: false,
+          value: '5'
+        }
+      ]
```

Comment:
vue que l'état initial de stepItems fait déjà ça, on pourrait faire:

```suggestion
      steps: state.ui.stepItems
```
pour n'avoir du code statique (et car si on prend les stepItems d'une autre source après et pas du state on peut suivre tout le code à éliminer plus facilement)

Reply:
Oui, je l'ai pas modifié parce que [la prochaine PR](https://github.com/CoorpAcademy/components/pull/2299/files#diff-0308bbfb1d95a7eca17e3d62d11182fdde3d4db1fa0c695ece4d3fbf7bf6dd7bR75) fait la dynamisation de ces valeurs. En gros, ici, j'ai voulu surtout me concentrer sur la partie components